src/AWS.Deploy.Recipes/RecipeDefinitions/ASP.NETAppElasticBeanstalkLinux.recipe

```diff
@@ -470,6 +470,15 @@
                     "DefaultValue": "Sun:00:00",
                     "AdvancedSetting": true,
                     "Updatable": true,
+                    "Validators": [
+                        {
+                            "ValidatorType": "Regex",
+                            "Configuration": {
+                                "Regex": "^(Mon|Tue|Wed|Thu|Fri|Sat|Sun):(0[0-9]|1\\d|2[0-3]):(0[0-9]|1\\d|2\\d|3\\d|4\\d|5\\d)$",
+                                "ValidationFailedMessage": "Invalid Preferred Start Time. You need to specify a day and time in the 'day:hour:minute' format."
```

Comment:
Also add an example of a valid start time in the `ValidationFailedMessage`.

Reply:
Done